test/extended/cluster/cl.go

```diff
@@ -74,11 +74,40 @@ var _ = g.Describe("[Feature:Performance][Serial][Slow] Load cluster", func() {
 				e2e.Logf("Our tuning set is: %v", tuning)
 			}
 			for j := 0; j < p.Number; j++ {
-				// Create namespaces as defined in Cluster Loader config
-				nsName := fmt.Sprintf("%s%d", p.Basename, j)
-				err := oc.Run("new-project").Args(nsName).Execute()
+				nsName := p.Basename
+				if len(p.IfExists) == 0 {
+					e2e.Logf("Parameter 'ifexists' not specified. Using 'default' value.")
+					p.IfExists = IF_EXISTS_DEFAULT
+				}
+				e2e.Logf("p.IfExists %v", p.IfExists)
+				if IF_EXISTS_REUSE != p.IfExists {
+					nsName = fmt.Sprintf("%s%d", p.Basename, j)
+				}
+				projectExists, err := ProjectExists(oc, nsName)
 				o.Expect(err).NotTo(o.HaveOccurred())
-				e2e.Logf("%d/%d : Created new namespace: %v", j+1, p.Number, nsName)
+				e2e.Logf("projectExists %v", projectExists)
+
+				if projectExists {
+					switch p.IfExists {
+					case IF_EXISTS_DEFAULT:
+						e2e.Failf("Project %v already exists. Use ifexists=reuse/delete in config", nsName)
+					case IF_EXISTS_REUSE:
+						e2e.Logf("reuse project %v", nsName)
+					case IF_EXISTS_DELETE:
+						err = DeleteProject(oc, nsName, true, 10*time.Second, 3*time.Minute)
+						o.Expect(err).NotTo(o.HaveOccurred())
+					default:
+						e2e.Failf("Unsupported ifexists value %v for project %v", p.IfExists, project)
+					}
+				}
+
+				if IF_EXISTS_REUSE != p.IfExists {
```

Comment:
Same as above, compare the variable to the value, not vice versa. This is also the third time you've checked for this condition.

Reply:
Done.
Now only switch/case on p.ifExists